pkg/datapath/linux/ipsec.go

```diff
@@ -369,16 +363,6 @@ func (n *linuxNodeHandler) enableIPSecIPv4Do(newNode *nodeTypes.Node, nodeID uin
 		statesUpdated = false
 	}
```

Comment:
Why do we still need this XFRM IN policy?

Reply:
This policy is just overlapping with the new "allow all" policies. 
We still need to do this since we do not expose methods directly for creating states. 

I've talked to you out of bound about this. We should follow up with a refactor which allows us to define and create states specifically, since we now have a single "allow all" IN policy.

Reply:
Oh, of course :facepalm: Somehow forgot these lines create both the state and policy. I clearly was still sick.